WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -158,7 +157,31 @@ private extension PersonViewController {
     }
 
     func removeWasPressed() {
-// TODO: JLP May.3.2016. To be implemented as part of #5288
+        let name = person.firstName?.nonEmptyString() ?? person.username
+        let title = NSLocalizedString("Remove User", comment: "Remove User Alert Title")
+        let message = NSLocalizedString("If you remove \(name), that user will no longer be able to " +
+                                        "access this site, but any content that was created by \(name) " +
+                                        "will remain on the site.\n\nWould you still like to remove this user?",
+                                        comment: "Remove User Confirmation Message")
```

Comment:
It would be nice to include a note about the significance `" \(name)"` syntax for our polyglots. For instance, we don't want to translate `name`.